app/models/post.rb

```diff
@@ -2,11 +2,15 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
   has_many :comments
   has_many :likes
-  def update_posts_counter
-    author.update(:posts_counter)
-  end
+  after_save :update_posts_counter
 
   def recent_comments
     comments.limit(5).order(created_at: :desc)
   end
+
+  private
+
+  def update_posts_counter
+    author.increment!(:posts_counter)
+  end
```

Comment:
- [ ] Here as well. I'm getting an error while creating `post` since you have used a different variable name here for `posts_counter` in comparison to the database schema. Please fix it.